src/js/classes/Table.js

```diff
@@ -101,6 +167,63 @@ class HeaderCell extends Cell {
 }
 
 
+class VizHeaderCell extends HeaderCell {
+  constructor(data, className, sortCol, sortDir, initSort, table, id) {
+    super(data, className, sortCol, sortDir, initSort, table, id);
+  }
+
+  render() {
+    const vizColors = ["green", "purple"];
+    const start = this.content["start"];
+    const end = this.content["end"];
+    const averages = this.content["averages"];
+
+    const cell = document.createElement("th");
+    cell.className = this.className;
+    // create start, end, and average tick/number elements
+    const startElement = this.createTickElement(start, "start-num");
+    const endElement = this.createTickElement(end, "end-num");
+    const averageElements = averages.map((average, i) => {
+      const text = `${average["name"]}:<br>${average["value"]}%`;
+      const className = "average";
+      return this.createTickElement(text, className, vizColors[i]);
+    });
+    // create wrapper around averages for positioning
+    const averageWrapper = document.createElement("div");
+    averageWrapper.className = "average-wrapper";
+    // offset the average elements by the value/end ratio (and subtract padding)
+    averageElements.forEach((element, i) => {
+      const left = averages[i]["value"] / end * 100;
+      const padding = averages.length > 1 ? "0.45em" : "2.05em";
```

Comment:
This is **super** nitpicky and something you don't have to fix right now but the "State Average" tick is like .1em to the left of the average line. Hopefully this isn't a difference in our two screens. The White/Black tics look correct on my screen.

Reply:
Fixed this! What I was doing before was super imprecise, but by adding some transforms, I was able to do this solidly all corresponding to hard-coded numbers.